### DISCUSSION TITLE
184 ~ 185 페이지 material-ui 적용 관련 질문입니다.

### DISCUSSION THREAD

**Discussion by doyeolKR** [fsoftwareengineer/todo-application#101] [질문과 답변]
안녕하세요 작가님 본 책으로 공부 잘 하고 있습니다 !
그런데 공부를 하던 중 질문 사항이 있어 글 남깁니다.

183 페이지에 
    Hello World 1
    Hello World 2
까지는 잘 따라하고 있었습니다. 

하지만 184 ~ 185 페이지의실습코드 
3-13. material ui를 이용한 Todo 컴포넌트 디자인, 
3-14. material ui를 이용한 App 컴포넌트 디자인의 코드를 작성하니 
메인화면에 아무것도 출력되지 않습니다.

몇 번이고 한 글자씩 확인해 보았지만 코드가 다른 부분은 없는 것 같습니다. 
왜 이런 현상이 발생하는 것일까요?

![image](https://user-images.githubusercontent.com/86040882/159452674-ef388524-0f6e-4211-994a-34fb6902e229.png)

![image](https://user-images.githubusercontent.com/86040882/159452972-0420a136-f70c-4f38-b429-934b053a0532.png)
![image](https://user-images.githubusercontent.com/86040882/159453103-7c460dff-3ce4-4850-8a2b-c3ef8d974131.png)

**Comment by fsoftwareengineer** [comment]
리액트나 mui의 버전 차이로 이런 현상이 발생 할 수 있습니다. 
https://github.com/fsoftwareengineer/todo-application/blob/main/6.4-Frontend_Deployment_With_AWS_ElasticBeantalk/todo-react-app/package.json
을 참고해 버전이 같은지 확인 해 주시기 바랍니다. 
버전을 업그레이드 하고 싶으시다면 구글에 mui와 리액트버전이 호환가능한지 검색해 주시기 바랍니다.
